requirements/base.txt

```diff
@@ -6,7 +6,10 @@
 git+https://github.com/edx/XBlock.git@fc5fea25c973ec66d8db63cf69a817ce624f5ef5#egg=XBlock
 git+https://github.com/edx/xblock-sdk.git@643900aadcb18aaeb7fe67271ca9dbf36e463ee6#egg=xblock-sdk
 
-edx-submissions>=0.0.8,<0.1.0
+# We are now forking the edx-submissions repo for ora score override function.
+#edx-submissions>=0.0.8,<0.1.0
```

Comment:
Let's remove this line instead of commenting out; we can find it in the log, if needed.